website/docs/plugins/nevergrad_sweeper.md

```diff
@@ -137,7 +135,29 @@ or a combination of both (eg.: `batch_size=log:int:4:1024`)
     is multiplicative. 
   - `integer`: set to `true` for integers (favor floats over integers whenever possible)
 
-  Providing only `lower` and `upper` bound will set the initial value to the middle of the range, and the step to a sixth of the range.
-
+Providing only `lower` and `upper` bound will set the initial value to the middle of the range, and the step to a sixth of the range.
 **Note**: unbounded scalars (scalars with no upper and/or lower bounds) can only be defined through a config file.
 
+### Defining through commandline override
```

Comment:
I think the section about command line should come before the section about config files.
Most users will be defining sweeps through the command line.

Reply:
sounds good, moved :)